code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -17,6 +17,7 @@
 	if(stat == UNCONSCIOUS)
 		if(is_zoomed)
 			zoom_out()
+		handle_critical_health_updates()
```

Comment:
This should only happen with xenos in crit, not with slept or disconnected ones.